test/skimNtuple2016.cpp

```diff
@@ -3661,7 +4038,7 @@ int main (int argc, char** argv)
           double chiA = tlv_firstLepton.M();  // hypothesised mass of invisible on side A.  Must be >=0.
           double chiB = tlv_secondLepton.M(); // hypothesised mass of invisible on side B.  Must be >=0.
 
-          // TES variations
+          /*// TES variations
```

Comment:
please do not use `\*...*\` for comments, put `\\` in front of each line, it' easier to read and correct in git.